### PR TITLE
Publish environment information to phone home.

### PR DESCRIPTION
**Pull request by kobalski**
New added items : 
- os.name
- os.arch
- os.version
- java.version
- java.vm.name

### CODE REVIEW

**Comment by mesutcelik** [comment]
👍

**Comment by mesutcelik** [comment]
I am ok with backporting this as well.

File: hazelcast/src/main/java/com/hazelcast/util/PhoneHome.java [hunk 1]
```diff
@@ -157,6 +158,11 @@ public String convertToLetter(int size) {
         parameterCreator.addParam("cjv", Integer.toString(clusterClientStats.get(ClientType.JAVA)));
         parameterCreator.addParam("cuptm", Long.toString(clusterUpTime));
         parameterCreator.addParam("nuptm", Long.toString(runtimeMxBean.getUptime()));
+        parameterCreator.addParam("osn", osMxBean.getName());
+        parameterCreator.addParam("osa", osMxBean.getArch());
+        parameterCreator.addParam("osv", osMxBean.getVersion());
+        parameterCreator.addParam("jvmn", runtimeMxBean.getVmName());
+        parameterCreator.addParam("jvmv", System.getProperty("java.version"));
```
Comment (emrahkocaman) [inline hazelcast/src/main/java/com/hazelcast/util/PhoneHome.java]: I think, we should surround this block with a try-catch as most of the `OperatingSystemMxBean` method can throw `SecurityException`. (https://docs.oracle.com/javase/7/docs/api/java/lang/management/OperatingSystemMXBean.html#getName())

**Comment by kobalski** [comment]
Backport is here : https://github.com/hazelcast/hazelcast/pull/7970

**Comment by emrahkocaman** [comment]
👍

**Comment by Serdaro** [comment]
The section http://docs.hazelcast.org/docs/3.6/manual/html-single/index.html#phone-home should be updated too?

**Comment by kobalski** [comment]
@Serdaro Yes. We should update docs too.

**Comment by Serdaro** [comment]
Updated the Preface chapter with the new fields for master and maintenance reference manuals.

**Pull request merged**